linode_api4/linode_client.py

```diff
@@ -1433,6 +1433,620 @@ def cancel(self):
         self.client.post("/object-storage/cancel", data={})
         return True
 
+    def transfer(self):
+        """
+        The amount of outbound data transfer used by your account’s Object Storage buckets,
+        in bytes, for the current month’s billing cycle. Object Storage adds 1 terabyte
+        of outbound data transfer to your data transfer pool.
+
+        API Documentation: https://www.linode.com/docs/api/object-storage/#object-storage-transfer-view
+
+        :returns: The amount of outbound data transfer used by your account’s Object
+                  Storage buckets, in bytes, for the current month’s billing cycle.
+        :rtype: integer
+        """
+        result = self.client.get("/object-storage/transfer")
+
+        if not "used" in result:
+            raise UnexpectedResponseError(
+                "Unexpected response when getting Transfer Pool!"
+            )
+
+        return MappedObject(**result)
+
+    def buckets(self, *filters):
+        """
+        Returns a paginated list of all Object Storage Buckets that you own.
+        This endpoint is available for convenience.
+        It is recommended that instead you use the more fully-featured S3 API directly.
+
+        API Documentation: https://www.linode.com/docs/api/object-storage/#object-storage-buckets-list
+
+        :returns: A list of Object Storage Buckets that matched the query.
+        :rtype: PaginatedList of ObjectStorageBucket
+        """
+        return self.client._get_and_filter(ObjectStorageBucket, *filters)
+
+    def bucket_create(self, cluster, label, acl="private", cors_enabled=False):
```

Comment:
Could we optionally accept an `ObjectStorageCluster` type here and for other methods that take in a cluster as an argument?